testsuite/tests/runtime-C-exceptions/stub_test.c

```diff
@@ -1,3 +1,4 @@
+#define _CRT_NONSTDC_NO_WARNINGS  /* for strdup */
```

Comment:
Nit, but can we put this define _after_ the other `#include`'s?

Reply:
No, it needs to be defined before including the system headers. It triggers whether the deprecation warning is attached to the function prototype.
Another option, that is found elsewhere in the code base, would be to write:
```c
#ifdef _WIN32
    return _strdup("bar");
#else
    return strdup("bar");
```

Reply:
OK, thanks. Since only tests are affected, I think the current solution is OK.